tests/v2/test_rln_group_manager_onchain.nim

```diff
@@ -154,7 +154,7 @@ proc stopGanache(runGanache: Process) {.used.} =
 
   # We wait the daemon to exit
   try:
-    let returnCodeExit = runGanache.waitForExit()
+    let returnCodeExit = runGanache.waitForExit(15)
```

Comment:
This is most likely the source of the error - https://nim-lang.org/docs/osproc.html#waitForExit%2CProcess%2Cint

The ganache instance is used more heavily in these tests, with more contract deployments and interactions, hence may have filled the output buffers. No reason for this to error out on ubuntu specifically though, so I may be wrong